corehq/apps/reports/standard/inspect.py

```diff
@@ -91,11 +91,10 @@ def es_query(self):
                                      lt=self.datespan.enddate_adjusted))
                  .filter(self._get_users_filter(mobile_user_and_group_slugs)))
 
-        # filter results by app and xmlns if applicable
-        if FormsByApplicationFilter.has_selections(self.request):
```

Comment:
Could delete the method `has_selections` altogether since this is the only use case for it.